pkg/dns/aws/dns.go

```diff
@@ -157,6 +154,14 @@ func NewProvider(config Config, operatorReleaseVersion string) (*Provider, error
 		}
 	}
 
+	// When RoleARN is provided, make a copy of the Route53 session and configure it to use RoleARN.
+	// RoleARN is intended to only provide access to another account's Route53 service, not for ELBs.
```

Comment:
```suggestion
	// RoleARN is intended to only provide access to another account's Route 53 service, not for ELBs.
```
Can you add a similar comment to the `RoleARN` field in the `Config` struct definition?

Reply:
Done.